claat/main.go

```diff
@@ -138,12 +139,16 @@ func usage() {
 
 const usageText = `Usage: claat <cmd> [export flags] src [src ...]
 
-Available commands are: export, update, version.
+Available commands are: export, serve, update, version.
 
 ## Export command
 
 Export takes one or more 'src' documents and converts them
-to the format specified with -f option.
+to the format specified with -f option. It takes no arguments
+and, with no path specified, it presents the current directory 
+contents. Clicking on a directory representing an exported
+codelab will load all the required dependencies and render
+the generated codelab as it would appear in production.
```

Comment:
@marcacohen I think you meant to put this down below, where `## Serve command` is, right?

Reply:
Yes, I saw that too and already fixed but thanks for flagging it!